pennylane/resource/__init__.py

```diff
@@ -15,7 +15,7 @@
 The ``resource`` module provides functionality to estimate the number of non-Clifford gates and
 logical qubits required to implement advanced quantum algorithms.
 """
-from .resource import Resources
+from .resource import Resources, count_resources

```

Comment:
IIRC, are we treating `count_resources` as a private / not user facing function?

Reply:
We can make it private.

Reply:
If tapes are user facing, is there any problem with making it user facing?

Reply:
Making this private 👍🏼